test/e2e/page-objects/pages/confirmations/redesign/confirmation.ts

```diff
@@ -11,7 +11,7 @@ class Confirmation {
 
   private headerAccountDetailsButton: RawLocator;
 
-  private footerCancelButton: RawLocator;
+  public footerCancelButton: RawLocator;
```

Comment:
:information_source:  I need to access this selector from the child class. I cannot use the clickFooterCancel method from this class because that waits for the window to close, and in the case of batch tx's, clicking this doesn't close the dialog, rather opens a new modal where we can then select the type of reject we want (cancel tx or cancel tx & upgrade)

Reply:
does it work if you change it to `protected` ?

Reply:
oh yeah, good call! I've update it :pray:

Reply:
:information_source:  the app has changed as now we use a different modal, so this change is not needed anymore